physicalplan/scan/vector_selector.go

```diff
@@ -113,6 +115,10 @@ func NewVectorSelector(pool *model.VectorPool, storage *seriesSelector, mint, ma
 	}
 }
 
+func (o *vectorSelector) Explain() (me string, next []model.VectorOperator) {
+	return fmt.Sprintf("[*vectorSelector] {%v} %v mod %v", o.storage.matchers, o.shard, o.numShards), nil
```

Comment:
Do we want to add storage parameters as well? Like mint, maxt, maybe selectors?

Reply:
We can add over time - do you think it will be useful for you?

Reply:
Adding selectors could get verbose, but it can be useful in cases like https://github.com/thanos-community/promql-engine/issues/66. Let's do it as a follow up when it's needed.